shared/output/gapic/templates/showcase/lib/google/showcase/v1beta1/echo/rest/service_stub.rb

```diff
@@ -172,7 +172,7 @@ def echo_error_details request_pb, options = nil
             # @yieldparam chunk [::String] The chunk of data received during server streaming.
             #
             # @return [::Gapic::Rest::TransportOperation]
-            def expand request_pb, options = nil, &block
+            def expand(request_pb, options = nil, &)
```

Comment:
Rubocop started making this change? I had no idea you could do this. (I guess Ruby 3.1+)